### DISCUSSION TITLE
My deployed filament is taking 5-6 seconds to load on each request

### DISCUSSION THREAD

**Discussion by ekstremedia** [filamentphp/filament#11519] [Help]
### Package

Other

### Package Version

v3.0.0

### How can we help you?

Hello! Im currently making a filament app, and development locally is running fine and is very fast. I was therefore suprised when i deployed it to my server, there it is taking 5-6 s to load on every page load. Ive been trying to debug this without any luck. Queries run super fast, everything seems to be in order but still it is slow. 

My code is this repo: https://github.com/ekstremedia/vefoto-filament

Even if i just create a empty route like: 

Route::get('/test', function () {
        return "TEST"; 
});

This is still taking many seconds to load. If i create a new blank laravel project on the same server, that loads super fast. Any ideas?

**Comment by leandrocfe** [comment]
maybe debugbar? Did you try to disable it?

**Reply by ekstremedia** [reply]
No, it has no effect. I left it on to show that no big amount of queries or models are loaded

**Reply by ArtMin96** [reply]
@ekstremedia Set views to false on debugbar config

**Reply by doender** [reply]
@ekstremedia I'm running into similar performance issues. Did you found a solution for this?

**Reply by ekstremedia** [reply]
@doender No, I could never figure this one out so unfortunately I had to scrap using filament entirely for my project.

**Comment by hizmarck** [comment]
@doender did you find something?

**Reply by doender** [reply]
We use Google Cloud Run and had 2 issues that made our app slow:
- Our database config included fetching a a key dynamically, and we were not caching properly (using `php artisan optimize`) so for every request we were fetching this key from Google Secret Manager
- On Cloud Run the default `opcache.memory_consumption` is quite low and we increased it to 256MB

Our application now takes around 650ms per request. Not perfect, but definitely usable for our use case.
